aiidalab_widgets_base/structures.py

```diff
@@ -842,15 +869,88 @@ def inner(ref, *args, **kwargs):
                 ref,
                 *args,
                 **kwargs,
-                atoms=ref.structure.copy(),
                 selection=ref.selection.copy(),
             )
 
     return inner
 
 
+class BasicCellEditor(ipw.VBox):
+    """Widget that allows for the basic cell editing."""
+
+    structure = Instance(Atoms, allow_none=True)
+
+    def __init__(self, title=""):
+        self.title = title
+
+        # cell transfor opration widget
+        primitive_cell = ipw.Button(
+            description="Convert to primitive cell",
+            layout={"width": "initial"},
+        )
+        primitive_cell.on_click(self.def_primitive_cell)
+
+        conventional_cell = ipw.Button(
+            description="Convert to conventional cell",
+            layout={"width": "initial"},
+        )
+        conventional_cell.on_click(self.def_conventional_cell)
+
+        super().__init__(
+            children=[
+                ipw.HTML(
+                    "<b>Cell transformation:</b>",
+                    layout={"margin": "20px 0px 10px 0px"},
+                ),
```

Comment:
I would argue this is a bit redundant. "Cell transformation" can also be set as a title.

Reply:
That's true, I use "Cell transform" for the default tab title and remove this HTML widget title.